src/index.js

```diff
@@ -229,15 +229,19 @@ class Router extends Component {
 		let current = active[0] || null;
 
 		let previous = this.previousUrl;
+		let previousRoute = this.previousRoute;
 		if (url!==previous) {
 			this.previousUrl = url;
+			this.previousRoute = current;
 			if (typeof onChange==='function') {
 				onChange({
 					router: this,
 					url,
 					previous,
+					previousRoute: previousRoute,
 					active,
-					current
+					current,
+					currentRoute: current
```

Comment:
Perhaps I'm misunderstanding, but what value does `currentRoute` here provide? Is it not just an alias for `current`?

Reply:
> You'll notice `currentRoute` is exactly the same as `current` - this is to offer a migration path in the future […]

As mentioned in my PR this is indeed an alias for `current`, to keep the API consistent. Otherwise you'd have 3 properties:
  - `previous` - a string URL
  - `current` - a Route instance
  - `previousRoute` - a Route instance

Including a `currentRoute` for parity means developers can reference `currentRoute` / `previousRoute` properties, which should be more intuitive than referencing `previous` / `currentRoute` which IMO would be pretty unclear. Does that make sense?

Reply:
I must've read over that, apologies.